lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -53,6 +53,24 @@ class LoadBalancer
 end
 end
 
+require 'chef/provider/load_balancer'
+class Chef
+class Provider
+  class LoadBalancer
+    # We override this so we can specify a machine name as `i-123456`
+    # This is totally a hack until we move away from base resources
+    def get_machine_spec!(machine_name)
```

Comment:
I would really like to get feedback on this - it directly fixes https://github.com/chef/chef-provisioning-aws/issues/247 but its a tooooooootal hack